sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/Swagger/mr-aoa.json

```diff
@@ -0,0 +1,451 @@
+{
```

Comment:
Don't forget to remove this and point `autorest.md` at the real thing.

Reply:
Will do after this round of iteration

Reply:
Autorest.md references checked in spec, swagger folder removed